README.md

```diff
@@ -35,21 +35,44 @@ echo $number; // echoes '123.456'
 ```php
 $a = new PrestaShop\Decimal\Number('123.456');
 $b = new PrestaShop\Decimal\Number('654.321');
+
 echo $a->plus($b); // echoes '777.777'
 ```
 
 ### Subtraction
 ```php
 $a = new PrestaShop\Decimal\Number('777.777');
 $b = new PrestaShop\Decimal\Number('654.321');
+
 echo $a->minus($b); // echoes '123.456'
 ```
 
+### Multiplication
+```php
+$a = new PrestaShop\Decimal\Number('777.777');
+$b = new PrestaShop\Decimal\Number('654.321');
+
+echo $a->times($b); // echoes '508915.824417'
+```
+
+### Division
+```php
+$a = new PrestaShop\Decimal\Number('777.777');
+$b = new PrestaShop\Decimal\Number('654.321');
+
+echo $a->dividedBy($b, 0);  // echoes '1'
+echo $a->dividedBy($b, 5);  // echoes '1.18867'
+echo $a->dividedBy($b, 10); // echoes '1.1886780341'
```

Comment:
The BC Math implementation doesn't round (see https://3v4l.org/l5Ob2) so I need to stick with their behavior.